vividus-tests/src/main/resources/properties/suite/additional/suite.properties

```diff
@@ -0,0 +1,2 @@
+bdd.story-loader.batch-1.resource-location=story/integration/additional
+bdd.story-loader.batch-1.resource-include-patterns=*.story
```

Comment:
```properties
bdd.story-loader.batch-4.resource-location=story/integration/additional
bdd.story-loader.batch-4.resource-include-patterns=*.story
```
otherwise the same properties from `integration` suite will override the properties from `additional` suite

Reply:
Done.